docs/library/network.WLAN.rst

```diff
@@ -133,4 +133,24 @@ Methods
    hostname       The hostname that will be sent to DHCP (STA interfaces) and mDNS (if supported, both STA and AP). (Deprecated, use :func:`network.hostname` instead)
    reconnects     Number of reconnect attempts to make (integer, 0=none, -1=unlimited)
    txpower        Maximum transmit power in dBm (integer or float)
+   pm             WiFi Power Management setting (see below for allowed values)
    =============  ===========
+
+Constants
+---------
+
+.. data:: WLAN.PM_PERFORMANCE
+        WLAN.PM_POWERSAVE
+        WLAN.PM_DEFAULT
+        WLAN.PM_NONE
+
+    Allowed values for the ``WLAN.config(pm=...)`` network interface parameter:
+
+        * ``PM_PERFORMANCE``: high performance wifi with higher power consumption
+        * ``PM_POWERSAVE``: reduce wifi power consumption with reduced
+          performance
```

Comment:
For performance it should be clear that it's still doing PM.  I suggest something like "enable power management but retain performance of the WiFi by only sleeping the RF components after a period of inactivity", or "balanced power management and WiFi performance"

For powersave, "enable maximum power saving capabilities at the expense of reduced WiFi performance".

Reply:
Good point. I hope I've respected your suggestion: 
- "enable WiFi power management to balance power savings and WiFi performance" and 
- "enable WiFi power management with additional power savings and reduced WiFi performance"
(leaving out any specifics about how the power saving is achieved - an inactivity timer is one way)